standard/statements.md

```diff
@@ -1096,6 +1096,34 @@ is then expanded to:
 
 The variable `e` is not visible to or accessible to the expression `x` or the embedded statement or any other source code of the program. The variable `v` is read-only in the embedded statement. If there is not an explicit conversion ([§10.3](conversions.md#103-explicit-conversions)) from `T` (the iteration type) to `V` (the *local_variable_type* in the `foreach` statement), an error is produced and no further steps are taken.
 
+When the iteration variable is a reference variable (§9.7), a `foreach` statement of the form
+
+```csharp
+foreach (ref V v in x) «embedded_statement»
+```
+
+is then expanded to:
```

Comment:
*[Also above on line 1077 but GitHub is refusing to allow me to comment on that line.]*

I don’t think the Standard is *requiring* this exact expansion. Elsewhere the Standard uses “equivalent to” or the longer “expanded to the semantic equivalent of” (see `try`).

For consistency I'd suggest we use “equivalent” here and on line 1077:

```suggestion
is then equivalent to:
```

Reply:
updated in both locations.